client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,98 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const bannerShape = {
+	siteId: PropTypes.number,
+};
+
+const bannerClickHandler = ( themeId, tracksEvent, { siteId, getBannerUrl, getCurrentTheme } ) => {
+	const url = getBannerUrl( themeId, siteId );
+	const tracksData = {
+		site_id: siteId,
+		current_theme: getCurrentTheme( siteId ),
+		timestamp: new Date().toGMTString(),
+	};
+	recordTracksEvent( tracksEvent, tracksData );
+	page( url );
```

Comment:
Can we avoid the `page` call and keep an `href` prop on `ThemesBanner`? (`href` and `action` aren't mutually exclusive e.g. for our `Button` component, so we can that behavior here I think.)